stripe/res/layout/select_shipping_method_widget.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+       android:layout_width="match_parent"
+       android:layout_height="match_parent"
+    >
+
+    <android.support.v7.widget.RecyclerView
```

Comment:
Any reason why not to just add a recyclerview to wherever you include this? Or is more planned to be here in future diffs?